proof_range.go

```diff
@@ -177,7 +177,11 @@ func (proof *KeyRangeProof) Verify(
 	// If we've reached this point, it means our range isn't empty, and we have
 	// a list of keys.
 	for i, path := range proof.PathToKeys {
-		leafNode := proofLeafNode{KeyBytes: keys[i], ValueBytes: values[i]}
+		leafNode := proofLeafNode{
+			KeyBytes:   keys[i],
+			ValueBytes: values[i],
+			Version:    proof.Version,
```

Comment:
Okay, version in leaf node proof is fair enough, as we do use in the hash, right?

Reply:
Yeah, it's part of the hash for leaves.